src/misc/lv_profiler_builtin.c

```diff
@@ -93,6 +93,8 @@ void lv_profiler_builtin_init(const lv_profiler_builtin_config_t * config)
         profiler_ctx.config.flush_cb("#\n");
     }
 
+    lv_profiler_builtin_set_enable(true);
```

Comment:
If you set it to `true` in init, then it's worth to set it `false` in uninit function.

Reply:
The structure will be [cleared](https://github.com/lvgl/lvgl/blob/master/src/misc/lv_profiler_builtin.c#L103) in `lv_profiler_builtin_uninit`, which will also set enable to false.